test/spec/protractor_sync_sherman_test.ts

```diff
@@ -0,0 +1,47 @@
+//TODO:  This is a temporary test file to be removed before merged.
+//TODO: The contents of this file should move ino README.md when ready
+import * as ab from 'asyncblock';
+
+import * as protractorSync from '../../app/index';
+import { browserSync, elementSync, polledExpect} from '../../app/index';
+
+protractorSync.configure({ implicitWaitMs: 500 });
+
+function createTest(fn: Function, errorMsg?: string) {
+  return (done: Function) => {
+    ab(() => {
+      fn();
+    }, (err: any) => {
+      if (errorMsg) {
+        expect(err.message).toEqual(errorMsg);
+      } else {
+        expect(err && err.stack || err || undefined).toBeUndefined();
+      }
+      done();
+    });
+  };
+}
+
+//TODO:  Ideally we have a test that highlights some of protractor-sync's awesome functionality.
+// I had an initial thought to emulate the subject matter in https://github.com/blackboard/protractor-sync/issues/62
+// but could be swayed to write a test against a different Web application
+//NOTE:  I was also uploaded on Chrome 65 and sendKeys was yielding this error:
+//    - Expected 'WebDriverError: unknown error: call function result missing 'value'
+//    - Updating Chromedriver to 2.36 solved this issue.
+describe('Google Translate', () => {
+  it('should have the Detect Language button focused by default', createTest(() => {
+    browserSync.get('https://translate.google.com/');
+    const googleTextForm = elementSync.findVisible('#gt-c');
+    polledExpect(() => googleTextForm.findVisible('#gt-lang-left .sl-sugg-button-container .jfk-button-checked')
+      .getText()).toEqual('Detect language');
```

Comment:
This is kinda nit-picky, but maybe we could do an example test that doesn't involve verifying text-on-page? It's not awful, it's just that it's not what we prefer to do in E2E tests in our own projects, and when protractor-sync makes its world debut this might not work for everyone. :)

Same thing goes for the test below this.

Reply:
right on.  totally can do.  I can brainstorm on other test ideas. :)
Actually, which reminds me of a follow-up question.
1.  other test ideas?  
2.  Should we go above and beyond providing a simple example or is that overkill?  
For example, the Protractor github page links to a [tutorial](http://www.protractortest.org/#/tutorial) that gives folks step by step guidance.

Reply:
1. Nothing clever. I think the Google Translate page is as good as any.
2. If you have time to build out a full tutorial I will certainly not try to stop you! I think that would be great! :) But if you don't have time for that now, then I think this change is a good step and we can take more steps later.

Reply:
cool, I'll take a quick gander hopefully later today with some new test tidbits.  If it's easy to add more robust steps into a tutorial i'll do.  Otherwise, I may take your advice :)